### PR TITLE
Add beep package for rt-thread.

### PR DESCRIPTION
**Pull request by Sunwancn**
添加了针对有源或无源蜂鸣器的鸣叫控制软件包。

### CODE REVIEW

**Comment by armink** [comment]
外层的 peripherals/Kconfig 也要记得修改下

File: peripherals/beep/Kconfig [hunk 1]
```diff
@@ -0,0 +1,82 @@
+
+menuconfig PKG_USING_BEEP
+    bool "beep: Buzzer beep package on rt-thread."
```
Comment (armink) [inline peripherals/beep/Kconfig]: 写个比较明确的软件包简介吧

Comment (Sunwancn) [reply peripherals/beep/Kconfig]: > 
> 
> 写个比较明确的软件包简介吧

第一次写package，这个简介要写在哪里？在Kconfig里吗？readme.md里是有写的。

Comment (armink) [reply peripherals/beep/Kconfig]: `beep: Buzzer beep package on rt-thread.` 指的是，这句简介太普通了

Comment (armink) [reply peripherals/beep/Kconfig]: 如果实在想到更好的，就这样也行

**Comment by Sunwancn** [comment]
好的，重写一下，谢谢了!还有，pull requests怎么close不掉？发自手机-------- 原始邮件 --------主题：Re: [RT-Thread/packages] Add beep package for rt-thread. (#584)发件人：朱天龙 收件人：RT-Thread/packages 抄送：Sunwancn ,Author 

@armink commented on this pull request.







In peripherals/beep/Kconfig:

> @@ -0,0 +1,82 @@

+

+menuconfig PKG_USING_BEEP

+    bool "beep: Buzzer beep package on rt-thread."



如果实在想不到更好的，就这样也行



—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

[

{

"@context": "http://schema.org",

"@type": "EmailMessage",

"potentialAction": {

"@type": "ViewAction",

"target": "https://github.com/RT-Thread/packages/pull/584#discussion_r394094328",

"url": "https://github.com/RT-Thread/packages/pull/584#discussion_r394094328",

"name": "View Pull Request"

},

"description": "View this Pull Request on GitHub",

"publisher": {

"@type": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Comment by armink** [comment]
底部有个 close 按钮，另外，简单写个一句话的简介就行，想不好就先算了

**Pull request closed without merging**